_Week8/group1/README.md

```diff
@@ -0,0 +1,32 @@
+# Group 1, Week 8 Assignment
+# Shaily Acharya, Sylvia Brown, Neel Desai
+
+
+## Overview
+
+Our simulation exercise simulates a sample dataset that can be used to estimate an individual's salary based on a variety of characteristics. In the sample model, the determinants of salary are GPA, undergraduate major, state in which their university is located, and university attended. Ultimately, the main outputs are 3 graphs that depict key characteristics of the dataset,  with the main graph depicting the number off clusters versus standard terror of the effect of GPA on salary. These graphs, along with a .csv file of the dataset, are included in this document. 
+
+## simulate.do
+
+The simulation do file defines the variables and the data generating processes that are behind our main output. 
+
+The parameter that we vary is the number of strata in our sample. In this case, the number of strata corresponds to the number of states that we are sampling from.  We generate the state-level fixed effects using a chi squared distribution— the rationale behind this is purely because we wanted to practice using a variety of different distributions in the randomization processes of our assignment. Then, we uniformly randomize the number of universities within each state, and generate the university-level fixed effects using a normal distribution. Lastly, we uniformly randomize the number of students per university.
+
+Once we define the relevant strata and fixed effects of our analysis, we continue on to generate the individual characteristics of the students. We generate variables for the parents' level of education (uniformly randomized), SAT score (randomized using a normal distribution), and AP credit (randomized using a Poisson distribution). Then, in order to generate the GPA variable that gets put in the regression for salary later in the program, we use the characteristics discussed above as well as a normally distributed randomization term. In order to generate university major-fixed effects, which also gets put into the regression for salary, we use uniform randomization. We define a local which represents the true coefficient on GPA, which we set as 5000. With these inputs, we then create the generative model for post-graduate salary on line 66, which is our main DGP. 
```

Comment:
Be clear about the relative scales and importance of the university vs state fixed effects. What is the function form of the GPA variable? What is the "true coefficient on GPA" -- is this the true increase in expected salary per grade point? What is the total range of possible salaries you will generate? Is $5,000 large or small relative to the total variation?